src/NuGet.Core/NuGet.Packaging/docs/README.md

```diff
@@ -0,0 +1 @@
+NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.
```

Comment:
Why did you choose to put READMEs inside of a docs folder? It may be helpful to have the READMEs directly in the project folder. Imagine I'm source diving the `NuGet.Protocol` codebase, I'd love to see a quick README in the `src/NuGet.Core/NuGet.Protocol` folder giving me a highlight of important types and whatnot.

Reply:
Thanks for reviewing!
Moved READMEs from /docs folder to project folder.
Fixed build errors after this moving.